### ISSUE TITLE
"forgot password" leads to a 500

### ISSUE THREAD

**Issue by bensigelman** [sourcegraph/sourcegraph.com#411]
(Not much to add beyond the issue title!)

**Comment by slimsag** [comment]
Thanks for reporting this! We'll do our best to get this fixed soon.

It looks like it works just fine for user accounts that exist (if you enter one that doesn't exist, however, it gives you the 500 error you are describing) -- so it looks like you might have entered an incorrect username?

**Comment by bensigelman** [comment]
Yeah, I think I had previously signed up using oauth... or maybe it was
part of the beta? I'm a friend of Quinn's and a coworker of Spoons who came
by your offices today. I was just going to kick the tires with one of our
private repos.

**Comment by nicot** [comment]
Hi Ben, you can sign up for an account at https://sourcegraph.com/join. We don't support private repos on sourcegraph.com right now.

Sorry for the password reset trouble, that'll be cleaned up soon and you should be able to reset the password even if you just remember your email address.
